### PR TITLE
Improve tower modules test playbooks

### PR DESCRIPTION
**Pull request by elyezer**
Update the playbooks so that they can be run multiple times against the
same instance.

### CODE REVIEW

File: tower_modules/tower_send/tasks/main.yml [hunk 1]
```diff
@@ -44,6 +44,11 @@
     that:
       - "result is changed"
 
+- name: Ensure the organization is not created
```
Comment (elyezer) [inline tower_modules/tower_send/tasks/main.yml]: Another solution would be trying to generate random organization names and remove them afterwards. I have no personal preference here just went for the quick route.

File: tower_modules/tower_settings/tasks/main.yml [hunk 2]
```diff
@@ -1,14 +1,36 @@
...
     name: "AWX_PROOT_SHOW_PATHS"
-    value: "'/var/lib/awx/projects/', '/tmp'"
+    value:
+      - '/var/lib/awx/projects/'
+      - '/tmp'
+  register: result
+
+- assert:
+    that:
+      - "result is changed"
```
Comment (elyezer) [inline tower_modules/tower_settings/tasks/main.yml]: In order to have changed as result we need to make sure that these values were not there. That is why we are reverting it on the next task. Another approach would be checking if the result is changed or ok.

File: tower_modules/tower_settings/tasks/main.yml [hunk 3]
```diff
@@ -1,14 +1,36 @@
...
   register: result
 
 - assert:
     that:
       - "result is changed"
 
 - name: Set the value of AWX_PROOT_BASE_PATH
+  tower_settings:
+    name: AWX_PROOT_BASE_PATH
+    value: "/tmp/newtmp"
```
Comment (elyezer) [inline tower_modules/tower_settings/tasks/main.yml]: The default value of this option is `/tmp` so we are setting it to another value to ensure it was actually changed.

**Comment by AlanCoding** [comment]
I think you have some good changes here. The only thing I'm uncertain about is the protocol for accepting such changes as we move forward with the transition to collections.

**Comment by AlanCoding** [comment]
https://galaxy.ansible.com/chrismeyersfsu/tower_modules

**Comment by elyezer** [comment]
@AlanCoding I am not sure about that too, I am just making sure that the tests that we are currently running are stable.

**Comment by elyezer** [comment]
Since I didn't hear anything against this I am going to merge it.

**Pull request merged**